rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,530 @@
+# Default directory mounts for the user's home directory
+homedir_default_mounts=".aws,.config,.emacs.d,.geodesic,.kube,.ssh,.terraform.d"
+
+function require_installed() {
+	if ! command -v $1 >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)